pieces/regex_json/python/main.py

```diff
@@ -1,3 +1,5 @@
+from __future__ import annotations
```

Comment:
Importing this will keep Python < 3.10 from erroring on types like `list` instead of `List`.